importlab/output.py

```diff
@@ -24,10 +24,7 @@ def format_file_node(import_graph, node, indent):
     if isinstance(f, resolve.Direct):
         out = '+ ' + f.short_path
     elif isinstance(f, resolve.Local):
-        out = os.path.relpath(f.path, f.fs.path)
-    elif isinstance(f, resolve.Relative):
-        # TODO(martindemello): Format depending on provenance[f.from_path]
-        out = '. ' + f.short_path
+        out = '  ' + os.path.relpath(f.path, f.fs.path)
```

Comment:
I'm seeing an attribute error on this line:
```
File "/usr/local/lib/python3.6/dist-packages/importlab-0.1.1-py3.6.egg/importlab/output.py", line 27, in format_file_node
    out = '  ' + os.path.relpath(f.path, f.fs.path)
AttributeError: 'OSFileSystem' object has no attribute 'path'
```

Reply:
fixed